### PR TITLE
Bump aide to 0.18.6

### PR DESCRIPTION
**Pull request by rakshaa2000**
###### Merge Checklist  

**All** boxes should be checked before merging the PR *(just tick any boxes which don't apply to this PR)*
- [x] The toolchain has been rebuilt successfully (or no changes were made to it)
- [x] The toolchain/worker package manifests are up-to-date
- [x] Any updated packages successfully build (or no packages were changed)
- [x] Packages depending on static components modified in this PR (Golang, `*-static` subpackages, etc.) have had their `Release` tag incremented.
- [x] Package tests (%check section) have been verified with RUN_CHECK=y for existing SPEC files, or added to new SPEC files
- [x] All package sources are available
- [x] cgmanifest files are up-to-date and sorted (`./cgmanifest.json`, `./toolkit/scripts/toolchain/cgmanifest.json`, `.github/workflows/cgmanifest.json`)
- [x] LICENSE-MAP files are up-to-date (`./SPECS/LICENSES-AND-NOTICES/data/licenses.json`, `./SPECS/LICENSES-AND-NOTICES/LICENSES-MAP.md`, `./SPECS/LICENSES-AND-NOTICES/LICENSE-EXCEPTIONS.PHOTON`)
- [x] All source files have up-to-date hashes in the `*.signatures.json` files
- [x] `sudo make go-tidy-all` and `sudo make go-test-coverage` pass
- [x] Documentation has been updated to match any changes to the build system
- [x] Ready to merge

---

###### Summary 

What does the PR accomplish, why was it needed?

###### Change Log  

- Bump aide to 0.18.6

###### Does this affect the toolchain?  

**NO**

###### Test Methodology

- Pipeline build id: [In Validation](https://dev.azure.com/mariner-org/mariner/_build/results?buildId=477181&view=results)

### CODE REVIEW

File: SPECS/aide/aide-0.15-syslog-format.patch [hunk 1]
```diff
@@ -1,496 +0,0 @@
-diff -up ./doc/aide.conf.5.in.syslog_format ./doc/aide.conf.5.in
```
Comment (mfrw) [inline SPECS/aide/aide-0.15-syslog-format.patch:1]: Are these patches already applied in 0.18.6 ?

File: SPECS/aide/aide.spec [hunk 2]
```diff
@@ -26,17 +26,17 @@ BuildRequires:  e2fsprogs-devel
...
-# fix aide in FIPS mode
-Patch2: aide-0.16b1-fipsfix.patch
-# Bug 1674637 - aide: FTBFS in Fedora rawhide/f30
-Patch3: aide-0.16-Use-LDADD-for-adding-curl-library-to-the-linker-comm.patch
-
-Patch4: aide-0.15-syslog-format.patch
-Patch5: aide-0.16-crypto-disable-haval-and-others.patch
-Patch6: coverity.patch
-Patch7: aide-0.16-crash-elf.patch
+# # Customize the database file location in the man page.
```
[the remark this answers is not in the document]
Comment (rakshaa2000) [reply SPECS/aide/aide.spec:29]: Sure

File: SPECS/aide/aide-0.15-syslog-format.patch (hunk 1 above)
Comment (rakshaa2000) [reply SPECS/aide/aide-0.15-syslog-format.patch:1]: No, I'll remove it. Seems like an oversight from my side

**Review by mfrw — APPROVED** [review]
LGTM

File: SPECS/aide/aide.spec [hunk 3]
```diff
@@ -26,17 +26,14 @@ BuildRequires:  e2fsprogs-devel
...
-# fix aide in FIPS mode
-Patch2: aide-0.16b1-fipsfix.patch
-# Bug 1674637 - aide: FTBFS in Fedora rawhide/f30
-Patch3: aide-0.16-Use-LDADD-for-adding-curl-library-to-the-linker-comm.patch
-
-Patch4: aide-0.15-syslog-format.patch
-Patch5: aide-0.16-crypto-disable-haval-and-others.patch
-Patch6: coverity.patch
-Patch7: aide-0.16-crash-elf.patch
+# Patch1: aide-0.16rc1-man.patch
```
Comment (mfrw) [inline SPECS/aide/aide.spec:29]: Please lets remove the comments regarding patch also

**Pull request merged**